core-test/src/main/java/org/openstack4j/api/network/LbPoolV2Tests.java

```diff
@@ -0,0 +1,100 @@
+import static org.testng.Assert.assertTrue;
+
+/**
+ *
+ * @author ashleykasim
+ *
+ */
+@Test(suiteName="Network/lbpoolv2", enabled=true)
+public class LbPoolV2Tests extends AbstractTest {
+    private static final String LBPOOLSV2_JSON = "/network/lbpoolsv2.json";
+    private static final String LBPOOLV2_JSON = "/network/lbpoolv2.json";
+    private static final String LBPOOLV2_UPDATE_JSON = "/network/lbpoolv2_update.json";
+
+    public void testListPoolV2() throws IOException {
+        respondWith(LBPOOLSV2_JSON);
+        List<? extends LbPoolV2> list = osv3().networking().lbaasV2().lbPoolV2().list();
+        assertEquals(2, list.size());
+        assertEquals(list.get(0).getId(), "b7f6a49f-ebd8-43c5-b792-5748366eff21");
+    }
+
+    public void testListPoolV2Filter() throws IOException {
+        respondWith(LBPOOLSV2_JSON);
+        Map<String, String> map = new HashMap<String, String>();
+        map.put("protocol", "HTTP");
+        List<? extends LbPoolV2> list = osv3().networking().lbaasV2().lbPoolV2().list(map);
+        assertEquals(2, list.size());
+    }
+
+    public void testGetPoolV2() throws IOException {
+        respondWith(LBPOOLV2_JSON);
+        String id = "b7f6a49f-ebd8-43c5-b792-5748366eff21";
+        LbPoolV2 pool = osv3().networking().lbaasV2().lbPoolV2().get(id);
+        assertNotNull(pool);
+        assertEquals(id, pool.getId());
+    }
+
+    public void testCreatePoolV2() throws IOException {
+        respondWith(LBPOOLV2_JSON);
+        String name = "testlbpool";
+        Protocol protocol = Protocol.HTTP;
+        LbPoolV2 create = Builders.lbpoolV2()
+                .adminStateUp(true)
+                .description("im a swimming pool")
+                .lbMethod(LbMethod.LEAST_CONNECTIONS)
+                .name(name)
+                .tenantId("6f759d84e3ca496ab77f8c0ffaa0311e")
+                .protocol(protocol)
+                .build();
+        LbPoolV2 result = osv3().networking().lbaasV2().lbPoolV2().create(create);
+        assertEquals(name, result.getName());
+        assertEquals(result.getLbMethod(), LbMethod.LEAST_CONNECTIONS);
+        assertEquals(protocol, result.getProtocol());
+    }
+
+    public void testUpdatePoolV2() throws IOException {
+        respondWith(LBPOOLV2_UPDATE_JSON);
+        String poolId = "b7f6a49f-ebd8-43c5-b792-5748366eff21";
+        String name = "v2update";
+        LbPoolV2Update update = Builders.lbPoolV2Update()
+                .adminStateUp(false)
+                .description("im a carpool")
+                .lbMethod(LbMethod.ROUND_ROBIN)
+                .name(name)
+                .build();
+        LbPoolV2 result = osv3().networking().lbaasV2().lbPoolV2().update(poolId, update);
+        assertEquals(name, result.getName());
+        assertEquals(LbMethod.ROUND_ROBIN, result.getLbMethod());
+        assertFalse(result.isAdminStateUp());
+    }
+
+    public void testDeletePoolV2() {
+        respondWith(200);
```

Comment:
yeah, but the response here doesn't check for 204 speciffically. Also, the v1 tests are checking for 200 on delete as well.